dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -60,6 +60,16 @@ kafka.library.not.present.CWMRX1006E=CWMRX1006E: Classes from the kafka-clients
 kafka.library.not.present.CWMRX1006E.explanation=Applications that use the Reactive Messaging Kafka Client must provide the kafka-clients JAR file and its dependencies. They can provide this file and its dependencies either by packaging it as a library within the application or by providing it as a shared library.
 kafka.library.not.present.CWMRX1006E.useraction=Package the kafka-clients JAR file and its dependencies as libraries in your application or as a shared library. The kafka-clients JAR file can be downloaded from the Apache Kafka website or from Maven Central.
 
+# Could not create a Kafka incoming connector for the {0} channel. The error is {1}
+kafka.create.incoming.error.CWMRX1007E=CWMRX1007E: Could not create a Kafka incoming connector for the {0} channel. The error is {1}
+kafka.create.incoming.error.CWMRX1007E.explanation=An error has occurred while setting up the Kafka connector for an incoming reactive messaging channel that prevents normal operation of MicroProfile Reactive Messaging.
```

Comment:
Change,
"An error has occurred while setting up the Kafka connector for an incoming reactive messaging channel that prevents normal operation of MicroProfile Reactive Messaging."
to,
"An error occurred while the Kafka connector was being set up for an incoming reactive messaging channel that prevents normal operation of MicroProfile Reactive Messaging."

Reply:
I suggest that "being" not be in the message. Thus, change the message to:

An error occurred while the Kafka connector was set up for an incoming reactive messaging channel that prevents normal operation of MicroProfile Reactive Messaging.

or:

An error that occurred when the Kafka connector was set up for an incoming reactive messaging channel prevents normal operation of MicroProfile Reactive Messaging.

Reply:
Used
"An error that occurred when the Kafka connector was set up for an incoming reactive messaging channel prevents normal operation of MicroProfile Reactive Messaging."